.github/workflows/reusable-release-policy-go.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and release a Kubewarden policy written in Go
+
+on:
+  workflow_call:
+    inputs:
+      oci-target:
+        type: string
+        required: true
+    secrets:
+      workflow-pat:
+        description: "Github Personal Access Token (PAT) that can trigger workflow-dispatch"
+        required: true
```

Comment:
I think you do not need this secret. AFAICS, this token is used to notify the policy hub. However, I believe you can use the $secrets.GITHUB_TOKEN$ for that as well. 
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow

Reply:
this is not possible in this case, we have to trigger an action on another repository. the github token has privileges only inside of the repo that owns the action.

In this case a policy needs to trigger an action inside of the policy-hub repository.

Reply:
I thought the same, but I think it cannot be :/. From https://github.com/marketplace/actions/workflow-dispatch#token, "NOTE. The automatically provided token e.g. ${{ secrets.GITHUB_TOKEN }} can not be used, GitHub prevents this token from being able to fire the workflow_dispatch".

But now I'm wondering if we are really doing a workflow_dispatch...

Reply:
we are, by triggering the dispatch via github's API

Reply:
Meerging then.

Reply:
:+1: